### ISSUE TITLE
Add 4 Team-play health stats to retro

### ISSUE THREAD

**Issue by jeffreywescott** [LearnersGuild/echo#499]
@shereefb commented on [Wed Oct 19 2016](https://github.com/LearnersGuild/game-prototype/issues/93)
## Benefits

Learners get more feedback on their team play skills. 
## Description

Add four more health stats (team play):

| Title | Question |
| :-- | :-- |
| Receptiveness | They were open to feedback, and received it gracefully without becoming defensive or dismissive. They separated their ego from their work. |
| Results Focus | They focused on the best outcomes for the team. They were dedicated to and persevered in achieving shared goals. |
| Flexible Leadership | They stepped into leadership roles when needed, and supported others to step into leading roles. |
| Friction Reduction | They were constantly looking for ways to improve team process, and to help everyone play to their full potential. |

Ideally these are asked right before the team play health stat (based on team play skills I would work with this person again)

These result of these questions should be reflected in the retro DM
#### Receptiveness

> Rate your agreement with the following statement:
> 
> "{{subject}} was open to feedback, and received it gracefully without becoming defensive or dismissive. {{subject}} separated their ego from their work."
#### Results Focus

> Rate your agreement with the following statement:
> 
> "{{subject}} focused on the best outcomes for the team. {{subject}} was dedicated to and persevered in achieving shared goals."
#### Flexible Leadership

> Rate your agreement with the following statement:
> 
> "{{subject}} stepped into leadership roles when needed, and supported others to step into leading roles."
#### Friction Reduction

> Rate your agreement with the following statement:
> 
> "{{subject}} was constantly looking for ways to improve team process, and to help everyone play to their full potential."

All of the above are of `responseType: likert7Agreement` and `subjectType: player`
### Extend post-retro message

``` js
`**RETROSPECTIVE RESULTS:** #${project.name}

**Feedback from your team:**
${teamFeedbackList.join('  \n\n')}

**Hours contributed:**
Team size: ${team.length}
Your hours: ${stats.hours || 0}
All team hours: ${stats.teamHours || 0}

${teamHoursList.join('  \n')}

**Contribution to the project:**
Self-assessed: ${stats.rcSelf || 0}%
Team-assessed: ${stats.rcOther || 0}%

Your estimated contribution to the project: ${stats.rc || 0}%
Expected contribution for # of hours: ${stats.ec || 0}%
Contribution difference: ${stats.ecd || 0}%

**Stats earned for this project:**
Learning Support: ${stats.ls || 0}%
Culture Contribution: ${stats.cc || 0}%
Team Play: ${stats.tp || 0}%
- Receptiveness: ${stats.recept || 0}%
- Results Focus: ${stats.resFocus || 0}%
- Flexible Leadership: ${stats.flexLead || 0}%
- Friction Reduction: ${stats.fricRedux || 0}%`
```

---

@bundacia commented on [Wed Oct 19 2016](https://github.com/LearnersGuild/game-prototype/issues/93#issuecomment-254915102)

👍 LGTM. 

One thing to note in terms of prioritization: 

There's an implementation risk here if we start work on this before https://github.com/LearnersGuild/game/issues/476 is complete, because I'm making changes to how questions are tied to stats as part of that issue. There could be some merge conflicts and/or a bit of rework if we start this in parallel.

---

@jeffreywescott commented on [Wed Oct 19 2016](https://github.com/LearnersGuild/game-prototype/issues/93#issuecomment-254929014)

Thanks, @bundacia. @tannerwelsh -- as soon as you finalize UI, I'll do final review and try to get it to the top of the Backlog on the implementation board ASAP.

---

@tannerwelsh commented on [Wed Oct 19 2016](https://github.com/LearnersGuild/game-prototype/issues/93#issuecomment-254944023)

UI updated. \cc @shereefb @jeffreywescott @bundacia @prattsj 

---

@shereefb commented on [Wed Oct 19 2016](https://github.com/LearnersGuild/game-prototype/issues/93#issuecomment-254945300)

lgtm!

---

@prattsj commented on [Wed Oct 19 2016](https://github.com/LearnersGuild/game-prototype/issues/93#issuecomment-254945546)

I like it. 👍 

---

@prattsj commented on [Wed Oct 19 2016](https://github.com/LearnersGuild/game-prototype/issues/93#issuecomment-254946606)

@tannerwelsh: should we update wording with this update (at least in the UI) to use the new health stats language?

```
.
.
.
**Health stats earned for this project:**
Technical: ${stats.ls || 0}%
Culture: ${stats.cc || 0}%
Team Play: ${stats.tp || 0}%
.
.
.
```

---

@tannerwelsh commented on [Thu Oct 20 2016](https://github.com/LearnersGuild/game-prototype/issues/93#issuecomment-255140501)

@prattsj moved that to its own issue: https://github.com/LearnersGuild/game/issues/498
